src/screen/CreateAccountScreen.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import {View, StyleSheet, TouchableOpacity} from 'react-native';
+import {Text, Input, ListItem, Icon, useTheme, Button} from '@ui-kitten/components';
+import {Keyring} from '@polkadot/keyring';
+import {mnemonicGenerate} from '@polkadot/util-crypto';
+import globalStyles, {monofontFamily, standardPadding} from 'src/styles';
+import FormLabel from 'presentational/FormLabel';
+import IdentityIcon from '@polkadot/reactnative-identicon/Identicon';
+import Padder from 'presentational/Padder';
+import Clipboard from '@react-native-community/clipboard';
+
+type Account = {
+  title: string;
+  password: string;
+};
+
+export function CreateAccountScreen() {
+  const theme = useTheme();
+  const keyring = new Keyring();
```

Comment:
I think we need `ss58Format` for distinguishing between networks. https://polkadot.js.org/docs/keyring/start/ss58/

I think polkadot account addresses are supposed to start with 1 at beginning.
```suggestion
  const keyring = new Keyring({type: 'sr25519', ss58Format: 0});
```

Reply:
We can add the `ss58Format` value in the networkContext for respective networks.

Reply:
Also I am not completely sure we need to be concerned about using type `sr25519` because of the following:
https://polkadot.js.org/docs/keyring/start/create/#revisiting-crypto

Reply:
I've added the ss58Format in network context on this pr https://github.com/litentry/litentry-app/pull/481/files